### PR TITLE
chore: Update kb0069.md

### PR DESCRIPTION
**Pull request by Nnyny**
change keyman version to the latest

### CODE REVIEW

File: knowledge-base/kb0069.md [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+When Adobe Acrobat Reader is open, you experience that you cannot type using a Keyman keyboard in the search box, the comment section nor the Fill & Sign. 
 
-<h1>Issue</h1>
-<p>The problem is that Adobe Acrobat Reader by default blocks all external applications from interaction, due to many longstanding security issues with it.</p>
+## Issue
+The problem is that Adobe Acrobat Reader by default blocks all external applications from interaction, due to many longstanding security issues with it.
 
-<h1>Resolution</h1>
+## Resolution
+Keyman Desktop 17 does not resolve the issue with Keyman keyboards not typing in Adobe Acrobat Reader. For a simple workaround, you can turn off Protected Mode in Adobe Acrobat Reader. Here's how:
```
Comment (mcdurdin) [inline knowledge-base/kb0069.md:10]: Keyman Desktop was renamed to Keyman for Windows in Keyman 14, so we need to say "Keyman for Windows" now

File: knowledge-base/kb0069.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 
 ## Applies to:
- * Keyman Desktop 12
- * Keyman Desktop 11
- * Keyman Desktop 10
- * Keyman Desktop 9
- * Keyman Desktop 8
+ * Keyman Desktop 10.0+
+ * Keyman Desktop 9.0
+ * Keyman Desktop 8.0
```
Comment (mcdurdin) [inline knowledge-base/kb0069.md:22]: I think we should say:

```suggestion
* Keyman for Windows 14.0+
* Keyman Desktop 8.0 - 13.0
```

**Review by mcdurdin — COMMENTED** [review]
This looks good so far, just fixup the Keyman for Windows name and we'll be good to go

File: knowledge-base/kb0069.md (hunk 1 above)
Comment (Nnyny) [reply knowledge-base/kb0069.md]: noted @mcdurdin

**Review by mcdurdin — APPROVED** [review]
LGTM!

**Pull request merged**